### PR TITLE
[KYUUBI #3131] Improve operation state change logging

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

closes https://github.com/apache/incubator-kyuubi/issues/3131

- use \n to replace , in operation state change logging
- only log statement if the state is running

### _How was this patch tested?_
before:
```
2022-07-25 14:03:31.391 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[183ee841-16b5-4444-a53f-3e8966e6af69]: INITIALIZED_STATE -> PENDING_STATE, statement: select 1
2022-07-25 14:03:31.410 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[183ee841-16b5-4444-a53f-3e8966e6af69]: PENDING_STATE -> RUNNING_STATE, statement: select 1
2022-07-25 14:03:31.433 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[183ee841-16b5-4444-a53f-3e8966e6af69]: RUNNING_STATE -> FINISHED_STATE, statement: select 1, time taken: 0.42 seconds
2022-07-25 14:03:31.446 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[183ee841-16b5-4444-a53f-3e8966e6af69]: FINISHED_STATE -> CLOSED_STATE, statement: select 1
```

after:
```
2022-07-25 18:47:34.870 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[c6a51097-5c38-4552-8c15-4b9466916d33]: PENDING_STATE -> RUNNING_STATE, statement:
select 1
2022-07-25 18:47:35.389 INFO org.apache.kyuubi.operation.ExecuteStatement: Processing anonymous's query[c6a51097-5c38-4552-8c15-4b9466916d33]: RUNNING_STATE -> FINISHED_STATE, time taken: 0.515 seconds
```

### CODE REVIEW

**Comment by yaooqinn** [comment]
> Why are the changes needed?

**Comment by ulysses-you** [comment]
> > Why are the changes needed?

@yaooqinn it's for help lookup the log, the statement will print several lines as the screen size is limited. The pr description can not show the issue that the markdown style will always use one line even if the string is long.

**Comment by yaooqinn** [comment]
can we print the statement once only in `case RUNNING` branch?

**Comment by ulysses-you** [comment]
I think we can, it's real noisy

File: kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala [hunk 1]
```diff
@@ -103,16 +103,18 @@ abstract class AbstractOperation(session: Session) extends Operation with Loggin
...
+      case RUNNING =>
+        info(s"Processing ${session.user}'s query[$statementId]: " +
+          s"${state.name} -> ${newState.name}, statement:\n$redactedStatement")
+        startTime = System.currentTimeMillis()
       case ERROR | FINISHED | CANCELED | TIMEOUT =>
         completedTime = System.currentTimeMillis()
-        timeCost = s", time taken: ${(completedTime - startTime) / 1000.0} seconds"
+        val timeCost = s"\ntime taken: ${(completedTime - startTime) / 1000.0} seconds"
+        info(s"Processing ${session.user}'s query[$statementId]: " +
+          s"${state.name} -> ${newState.name}$timeCost")
```
Comment (ulysses-you) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala:115]: for terminated state, simplify to log user, statementId and timeCost

File: kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala [hunk 2]
```diff
@@ -103,16 +103,18 @@ abstract class AbstractOperation(session: Session) extends Operation with Loggin
...
     newState match {
-      case RUNNING => startTime = System.currentTimeMillis()
+      case RUNNING =>
+        info(s"Processing ${session.user}'s query[$statementId]: " +
+          s"${state.name} -> ${newState.name}, statement:\n$redactedStatement")
+        startTime = System.currentTimeMillis()
       case ERROR | FINISHED | CANCELED | TIMEOUT =>
         completedTime = System.currentTimeMillis()
-        timeCost = s", time taken: ${(completedTime - startTime) / 1000.0} seconds"
+        val timeCost = s"\ntime taken: ${(completedTime - startTime) / 1000.0} seconds"
```
Comment (yaooqinn) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala:113]: is \n still necessary?

**Review by turboFei — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thank you for review, merging to master

**Pull request closed without merging**